src/main/java/at/hannibal2/skyhanni/utils/LocationUtils.kt

```diff
@@ -41,4 +43,6 @@ object LocationUtils {
         val inFov = true // TODO add Frustum "Frustum().isBoundingBoxInFrustum(entity.entityBoundingBox)"
         return noBlocks && notTooFar && inFov
     }
+
+    fun inAdvancedMiningIsland() = IslandType.DWARVEN_MINES.isInIsland() || IslandType.CRYSTAL_HOLLOWS.isInIsland()
```

Comment:
either move to lorenz utils, or move all isinisland, current area, current location, is on hypixel, etc checks all into this class. but this doesnt match the scope of the pr i thnk.